hicommonwealth/commonwealth
# Use active address, not last fetched on thread creation

## Link to Issue
Closes: #4046 

## Description of Changes
- Uses active address upon submission of new thread, rather than last updated author address. 
   - Prev current author was fetched from state on each keystroke so if address is change and no more watched events occur new address will not be used 

## Test Plan
- Go to create Thread
- type something 
- switch to other address w/o typing more 
- create thread and ensure correct address was used
